app/templates/community/meetups/index.hbs

```diff
@@ -4,38 +4,38 @@
 
   <section aria-labelledby="meetups" class="mb-3">
     <h2 id="meetups">Find a meetup in your city</h2>
-    {{#if leafletPackageLoaded}}
-      {{#leaflet-map class="leaflet-container" lat=lat lng=lng zoom=zoom}}
-        {{tile-layer url="https://{s}.basemaps.cartocdn.com/light_all/{z}/{x}/{y}.png"}}
+    {{#if this.leafletPackageLoaded}}
+      <LeafletMap @class="leaflet-container" @lat={{this.lat}} @lng={{this.lng}} @zoom={{this.zoom}}>
+        <TileLayer @url="https://{s}.basemaps.cartocdn.com/light_all/{z}/{x}/{y}.png" />
 
-        {{#each model as |meetup|}}
-          {{#marker-layer lat=meetup.lat lng=meetup.lng}}
-            {{#popup-layer}}
+        {{#each this.model as |meetup|}}
```

Comment:
**suggestion (blocking):**

Here, `model` came from the route's model hook so we want to use `@model`. (We can also guess this because the controller file doesn't exist.)

```suggestion
        {{#each @model as |meetup|}}
```